CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
 
+## 8.6.6 - September 25, 2020
```

Comment:
Previous version was `9.5.0-beta.1` and now it's `8.6.6`? 
It's a bit strange I guess 🤔

Reply:
This is a patch release based on 8.6.5 for only one fix.